surfactant/cmd/stat.py

```diff
@@ -1,6 +1,49 @@
 import click
 
+from surfactant.sbomtypes import SBOM
+
 
 @click.command("stat")
-def stat():
+@click.argument("input_sbom", type=click.File("r"), required=True)
+def stat(input_sbom):
     click.echo("Running stat command")
+    data = SBOM.from_json(input_sbom.read())
+    elfIsLib = 0
+    elfIsExe = 0
+    peIsExe = 0
+    peIsdll = 0
+    clrExe = 0
+    clrDll = 0
+    sw = data.software
+    for i in range(len(sw)):
+        md = sw[i].metadata
+        for n in range(len(md)):
+            if "elfIsLib" in md[n]:
+                if md[n]["elfIsLib"]:
+                    elfIsLib += 1
+            if "elfIsExe" in md[n]:
+                if md[n]["elfIsExe"]:
+                    elfIsExe += 1
+            if "peIsExe" in md[n]:
+                if md[n]["peIsExe"]:
+                    peIsExe += 1
+            if "peIsDll" in md[n]:
+                if md[n]["peIsDll"]:
+                    peIsdll += 1
+            if "peIsClr" in md[n]:
+                if md[n]["peisClr"]:
+                    if md[n]["peIsExe"]:
+                        clrExe += 1
+                    else:
+                        clrDll += 1
```

Comment:
```suggestion
    for sw in data.software:
        for md in sw.metadata:
            if "elfIsLib" in md:
                if md["elfIsLib"]:
                    elfIsLib += 1
            if "elfIsExe" in md:
                if md["elfIsExe"]:
                    elfIsExe += 1
            if "peIsExe" in md:
                if md["peIsExe"]:
                    peIsExe += 1
            if "peIsDll" in md:
                if md["peIsDll"]:
                    peIsdll += 1
            if "peIsClr" in md:
                if md["peisClr"]:
                    if md["peIsExe"]:
                        clrExe += 1
                    else:
                        clrDll += 1
```

Fixes pre-commit errors from pylint: `C0200: Consider using enumerate instead of iterating with range and len (consider-using-enumerate)`